src/style.css

```
@@ -1 +1,57 @@
-/* Styles go here */
+body,
+.nav__list {
+  margin: 0;
+}
```

reset is done for the tags. this is the only time it is acceptable to style tags, so you don't have to do it for every `ul` individually
```suggestion
body,
ul {
  margin: 0;
}
```